doc/md/fundamentals/1-basic-syntax/9-operators.md

```diff
@@ -201,9 +201,8 @@ func eval(e : Exp) : ? Nat {
 }
 ```
 
-To guard against division by 0 without trapping, the eval function returns an option result, using null to indicate failure.
-
-Each recursive call is checked for null using !, immediately exiting the outer do ? block, and then the function itself, when a result is null.
+To guard against division by 0 without trapping, the eval function returns an option result, using `null` to indicate failure.
 
+Each recursive call is checked for `null` using `!`, immediately exiting the outer `do ?` block, and then the function itself, when a result is `null`.
 
```

Comment:
This section doesn't discuss and or and not and the fact that or and and are short-circuiting